specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/2022-08-01/availabilitySet.json

```diff
@@ -0,0 +1,514 @@
+          "nextLinkName": null
+        },
+        "x-ms-examples": {
+          "AvailabilitySets_ListAvailableSizes_MaximumSet_Gen": {
+            "$ref": "./examples/availabilitySetExamples/AvailabilitySets_ListAvailableSizes_MaximumSet_Gen.json"
+          },
+          "AvailabilitySets_ListAvailableSizes_MinimumSet_Gen": {
+            "$ref": "./examples/availabilitySetExamples/AvailabilitySets_ListAvailableSizes_MinimumSet_Gen.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "AvailabilitySetProperties": {
+      "properties": {
+        "platformUpdateDomainCount": {
+          "type": "integer",
+          "format": "int32",
+          "description": "Update Domain count."
+        },
+        "platformFaultDomainCount": {
+          "type": "integer",
+          "format": "int32",
+          "description": "Fault Domain count."
+        },
+        "virtualMachines": {
+          "type": "array",
+          "items": {
+            "$ref": "../../../common-types/v1/common.json#/definitions/SubResource"
+          },
+          "description": "A list of references to all virtual machines in the availability set."
+        },
+        "proximityPlacementGroup": {
+          "$ref": "../../../common-types/v1/common.json#/definitions/SubResource",
+          "description": "Specifies information about the proximity placement group that the availability set should be assigned to. <br><br>Minimum api-version: 2018-04-01."
+        },
+        "statuses": {
+          "readOnly": true,
+          "type": "array",
+          "items": {
+            "$ref": "./computeRPCommon.json#/definitions/InstanceViewStatus"
+          },
+          "x-ms-identifiers": [],
+          "description": "The resource status information."
+        }
+      },
+      "description": "The instance view of a resource."
+    },
+    "AvailabilitySetSkuType": {
+      "type": "string",
+      "description": "Specifies the sku of an Availability Set. Use 'Aligned' for virtual machines with managed disks and 'Classic' for virtual machines with unmanaged disks. Default value is 'Classic'.",
+      "enum": [
+        "Classic",
+        "Aligned"
+      ],
+      "x-ms-enum": {
+        "name": "AvailabilitySetSkuTypes",
+        "modelAsString": true
+      }
+    },
+    "AvailabilitySet": {
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/AvailabilitySetProperties"
+        },
+        "sku": {
+          "$ref": "./computeRPCommon.json#/definitions/Sku",
+          "description": "Sku of the availability set, only name is required to be set. See AvailabilitySetSkuTypes for possible set of values. Use 'Aligned' for virtual machines with managed disks and 'Classic' for virtual machines with unmanaged disks. Default value is 'Classic'."
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "../../../common-types/v1/common.json#/definitions/Resource"
```

Comment:
>/common-types/ [](http://example.com/codeflow?start=27&length=14)

It does not look like this common-types folder is included in this PR .. dont you need that one as well ?

Also is there any reason you are not using the actual repo common types ? [https://github.com/Azure/azure-rest-api-specs/blob/feature/cplat-2022-08-01-revised/specification/common-types/resource-management/v3/types.json](https://github.com/Azure/azure-rest-api-specs/blob/feature/cplat-2022-08-01-revised/specification/common-types/resource-management/v3/types.json)

Reply:
Thank you for your review @raosuhas 

it is included in this PR, you can see here: https://github.com/Azure/azure-rest-api-specs/pull/20552/files#diff-8a700e636a2b2f30c29b236ca3ef91d52850d0eab5a76273e593046e397babb7

so this one is referring to specification/compute/resource-manager/Microsoft.Compute/common-types/ which has objects that are used across different RPs in Compute Service,   
the other common-types,  /specifications/common-types has objects that are used across different services.

Reply:
@raosuhas  pls review !